src/devices/plic.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+enum PLIC_REG {
+    INTR_PENDING = 0x400,
+    INTR_ENABLE = 0x800,
+    INTR_PRIORITY = 0x80000,
```

Comment:
Change the name to `INTR_PRIO_THR` or `INTR_THRESHOLD` should be more clear reference to the plic spec.